tests/test_control_realtime.py

```diff
@@ -54,41 +55,44 @@ class TestControlInterfaceRealtime(unittest.TestCase):
     the transfered data so it can be compared to the expected traffic.
     """
 
+    def setUp(self):
+        self.host = os.getenv("XLED_TEST_HOST", "192.168.10.100")
+        self.numleds = int(os.getenv("XLED_TEST_NUMBER_OF_LED", "250"))
+
     def redirect_xled_socket_to_fake_client(self, ctr):
         self.fakeclient = FakeUDPclient()
         ctr._udpclient = self.fakeclient
 
     @vcr.use_cassette("tests/cassettes/TestControlInterface.test_realtime.yaml")
     def test_realtime_protocols(self):
-        num_leds = 250
         ctr = ControlInterface("192.168.10.100")
```

Comment:
@scrool, Nice adjustment of all the tests! Makes it much more convenient.

But note that "test_control_realtime" was the very first tests I wrote, and it is not as adapted to this kind of testing as the other test files. So I had not moved out the IP-number yet. This line should use self.host.

Reply:
Ah, right. Good catch! It should be fixed now.